setup.py

```diff
@@ -98,7 +98,7 @@ def build_extension(self, ext):
 
 setup(
     name='sagemaker_tensorflow',
-    version='2.6.0.2.0.0',
+    version='2.6.0.1.11.0',
```

Comment:
Why is this "downgraded"?

Reply:
My exact question. Looks like the earlier version bump to 2x happened mistakenly and this is the correct ordering. cc @ahsan-z-khan

Reply:
yes +1 to what @shreyapandit  said